Client/webpack.config.js

```diff
@@ -1,24 +1,33 @@
-import path from 'path';
```

Comment:
I've only learned this through hours of debugging with Clients. You are better off not using import in webpack files. It *can* work if you name the file `webpack.babel.config.js`. The problem is a webpack config file is an Entry file to start processing other files. Babel won't have run yet - so `import` statements aren't available to be transpiled down to `requires`